app/models/book.rb

```diff
@@ -4,4 +4,20 @@ class Book < ActiveRecord::Base
   def has_summary?
     summary.present?
   end
+
+  def self.by_date
+    order(created_at: :desc)
+  end
+
+  def self.by_title
+    order('lower(title)')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.